contrib/babelfishpg_tsql/src/iterative_exec.c

```diff
@@ -99,44 +99,75 @@ static int exec_stmt_raiserror(PLtsql_execstate *estate, PLtsql_stmt_raiserror *
 	int		severity = -1;
 	int		state = -1;
 
-	/* Read parameters of RAISERROR statements */
-	read_raiserror_params(estate, stmt->params, stmt->paramno, &msg, &msg_id, &severity, &state);
-	msg = pstrdup(msg);
+	PLtsql_expr *temp_expr;
```

Comment:
Let's create another function to handle `pltsql_explain_only` and call it here. In that way, we can make the original function simple and reduce unnecessary diffs.